schema/deploy/tables/attachment.sql

```diff
@@ -0,0 +1,69 @@
+-- Deploy ggircs-portal:tables/attachment to pg
+
+begin;
+
+create table ggircs_portal.attachment
+(
+  id integer primary key generated always as identity,
+  file uuid,
+  description varchar(10000),
+  file_name varchar(1000),
+  file_type varchar(100),
+  file_size varchar(100),
+  application_id integer not null references ggircs_portal.application(id),
+  version_number int,
+  foreign key (application_id, version_number) references ggircs_portal.application_revision(application_id, version_number),
+  unique (application_id, version_number)
+);
+
+create index attachment_file on ggircs_portal.attachment(file);
+create index application_id on ggircs_portal.attachment(application_id);
+create index ggircs_portal_attachment_foreign_key on ggircs_portal.attachment(application_id, version_number);
+
+select ggircs_portal_private.upsert_timestamp_columns(
+  table_schema_name := 'ggircs_portal',
+  table_name := 'attachment',
+  add_create := true,
+  add_update := true,
+  add_delete := false);
```

Comment:
**To be done in a separate PR:**
We will need to make `add_delete` true here to enable soft delete

Reply:
(And add comments on the new `delete` columns)